src/app/CommandHandler.cpp

```diff
@@ -116,6 +116,11 @@ CHIP_ERROR CommandHandler::ProcessCommandDataElement(CommandDataElement::Parser
         // Currently, it could be failed to decode Path or failed to find cluster / command on desired endpoint.
         // TODO: The behavior when receiving a malformed message is not clear in the Spec. (Spec#3259)
         // TODO: The error code should be updated after #7072 added error codes required by IM.
+        if (err == CHIP_ERROR_INVALID_PROFILE_ID)
+        {
+            ChipLogDetail(DataManagement, "Cannot found cluster 0x%" PRIx16 " on endpoint 0x%" PRIx8, clusterId, endpointId);
```

Comment:
I believe there is a typo here:
`found` - suggest `Cannot find cluster ...` or `No cluster ...` (slightly smaller log print)

Reply:
Thanks!